services/gateway/pkg/config/config.go

```diff
@@ -85,7 +85,7 @@ type StorageRegistry struct {
 
 // Cache holds cache config
 type Cache struct {
-	StatCacheStore          string        `yaml:"stat_cache_store" env:"OCIS_CACHE_STORE;GATEWAY_STAT_CACHE_STORE" desc:"The type of the cache store. Supported values are: 'memory', 'ocmem', 'etcd', 'redis', 'redis-sentinel', 'nats-js', 'noop'. See the text description for details."`
+	StatCacheStore          string        // NOTE: The stat cache is not working atm. Hence we block configuring it
```

Comment:
@kobergj Note that this also needs a doc change, see:
[admin docs](https://doc.owncloud.com/ocis/next/deployment/services/s-list/gateway.html#caching) and [dev-docs](https://owncloud.dev/services/gateway/#caching). Pls update the gateway readme accordingly, I will take care on admin docs afterwards.

Reply:
Issue tackled here: https://github.com/owncloud/ocis/pull/7977